js/src/components/tree-select-control/index.js

```diff
@@ -54,7 +56,10 @@ const TreeSelectControl = ( {
 	let instanceId = useInstanceId( TreeSelectControl );
 	instanceId = id ?? instanceId;
 	const [ isExpanded, setIsExpanded ] = useState( false );
-	const optionsRef = useIsEqualRefValue( options );
+	const optionsRef = useIsEqualRefValue( [
+		{ value: 'all', label: selectAllLabel, children: options },
+	] );
```

Comment:
❓ What would happen if there is already an option with the value `"all"` in `options`? I think it wouldn't work as expected?